daisy_workflows/export/disk_export.wf.json

```diff
@@ -35,29 +35,40 @@
     }
   },
   "Sources": {
-    "${NAME}_export_disk.sh": "./export_disk.sh"
+    "${NAME}_export_disk.sh": "./export_disk.sh",
+    "${NAME}_disk_resizing_mon.sh": "./disk_resizing_mon.sh"
```

Comment:
Where is this script coming from?

Reply:
it's the existing one, named "disk_resizing_mon.sh" under "daisy_workflows_export"